test/CMakeLists.txt

```diff
@@ -60,57 +52,107 @@ else()
-endif()
+	set( CRTM_COEFFS_BRANCH_PREFIX "crtm" )  
+  set( CRTM_COEFFS_BRANCH "3.0.1_skylab_6.0" ) 
+
+#  set( DOWNLOAD_BASE_URL "https://bin.ssec.wisc.edu/pub/s4/CRTM/file")
+#
+#  set( test_files_dirname crtm_coefficients_${CRTM_COEFFS_BRANCH}.tar.gz )  #UPDATE TARBALL FILENAME HERE
+#  set( checksum "9cf4b1cc57e20802e608a597d7f53073") #MD5SUM, update this if the file above is updated / changed
+#
+#  message(STATUS "fix/ does not exist, download CRTM coeffs files from: ${DOWNLOAD_BASE_URL} to ${CRTM_COEFFS_PATH}")
+#
+#  list( APPEND CRTM_DATA_DOWNLOADER_ARGS
+#                ${DOWNLOAD_BASE_URL}
+#                ${CRTM_COEFFS_PATH}
+#                ${test_files_dirname}
+#                ${checksum} )
+#
+#  # Create download script for ufo_get_*_test_data test
+#  set ( FILENAME crtm_data_downloader.py)
+#  set ( SOURCE_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${FILENAME} )
+#  set ( DEST_FILE ${CMAKE_BINARY_DIR}/bin/${FILENAME} )
+#
+#  if( EXISTS "${SOURCE_FILE}.in" )
+#    configure_file( ${SOURCE_FILE}.in ${DEST_FILE} @ONLY )
+#  else()
+#    configure_file( ${SOURCE_FILE}    ${DEST_FILE} @ONLY )
+#  endif()
+#
+#  # add command to download coefficient data during build step, rather than as a test
+#  add_custom_target(get_crtm_coeffs
+#    COMMAND ${CMAKE_BINARY_DIR}/bin/crtm_data_downloader.py ${CRTM_DATA_DOWNLOADER_ARGS})
+
+  # Existing definitions
+  set(CRTM_COEFFS_PATH ${CMAKE_BINARY_DIR}/test_data/${REPO_VERSION})
+  file(MAKE_DIRECTORY ${CRTM_COEFFS_PATH})
+
+  set(DOWNLOAD_BASE_URL "https://bin.ssec.wisc.edu/pub/s4/CRTM/file")
+  set(test_files_dirname crtm_coefficients_${CRTM_COEFFS_BRANCH}.tar.gz) 
+  set(checksum "5f6d1c7c272af5a2ab39c72235bcd2a8") # MD5SUM
+
+  set(DOWNLOAD_URL "${DOWNLOAD_BASE_URL}/${test_files_dirname}")
+  set(DOWNLOAD_DEST "${CRTM_COEFFS_PATH}/${test_files_dirname}")
+
+  # Download during CMake configure step
+  message(STATUS "Downloading CRTM coeffs files from: ${DOWNLOAD_URL} to ${DOWNLOAD_DEST}")
+  file(DOWNLOAD ${DOWNLOAD_URL} ${DOWNLOAD_DEST}
+       EXPECTED_MD5 ${checksum}
+       SHOW_PROGRESS)
+
+  # -- UNTAR -- 
+
+  # Define the directory to untar into
+  set(UNTAR_DEST "${CMAKE_BINARY_DIR}/test_data/${REPO_VERSION}")
+
+  # Only untar if the CHECK_PATH does not exist
+  if(NOT EXISTS ${UNTAR_DEST}/})
+    execute_process(COMMAND tar -xzf ${DOWNLOAD_DEST} -C ${UNTAR_DEST}
+                    RESULT_VARIABLE result
+                    OUTPUT_QUIET ERROR_QUIET)
+    if(NOT result EQUAL 0)
+      message(FATAL_ERROR "Failed to untar the file.")
+    endif()
+  else()
+    message(STATUS "Files have already been untarred. Skipping.")
+  endif()
 
-# Add files to cmake resources
-ecbuild_add_resources( TARGET crtm_test_scripts
-                       SOURCES_PACK
-                       ${crtm_test_input}
-                     )
+
+endif()
 
 # Create list of sensor ids for testing
 list( APPEND Simple_Sensor_Ids
         atms_n21
         cris-fsr_n21
         v.abi_g18
-	atms_npp
+				atms_npp
```

Comment:
Are these indentation changes intentional?

Reply:
I think it's just a tabs vs. spaces thing.   It's not intentional.  I have converted all tabs to spaces just to be consistent. (`alt-x` `untabify` in emacs).